wnprc_billing/src/org/labkey/wnprc_billing/WNPRC_BillingController.java

```diff
@@ -678,4 +688,43 @@ private PDFFile getInvoicePDF(String invoiceNumber, String formName) throws IOEx
         pdfFile.setInvoicePDF(pdf);
         return pdfFile;
     }
+
+    //This function returns the current container's values for the 'CreditToAccount' and 'JetCsvSetting' variables.
+    @ActionNames("getProgramIncomeAccount")
+    @RequiresNoPermission
+    public class GetProgramIncomeAccountAction extends ReadOnlyApiAction<Object>
+    {
+        @Override
+        public ApiResponse execute(Object form, BindException errors) throws Exception
+        {
+            //Gets current values of CreditToAccount and JetCsvSetting.
+            Map<String, ModuleProperty> moduleProperties = ModuleLoader.getInstance().getModule(WNPRC_BillingModule.class).getModuleProperties();
+            var currentCreditToAccount = moduleProperties.get(WNPRC_BillingModule.CreditToAccount).getEffectiveValue(getContainer());
```

Comment:
this is fascinating. I didn't know you could use "var" to just auto detect a type. pretty cool

Reply:
😎